celery/app/base.py

```diff
@@ -471,6 +477,18 @@ def run(*args, **kwargs):
                     try:
                         return task._orig_run(*args, **kwargs)
                     except autoretry_for as exc:
+                        if retry_backoff:
```

Comment:
You're certainly welcome to check out my code from this branch and manually test it, if that's what you mean. If you're asking to change the code in this branch, I'd prefer if you made a new branch off of this one, make whatever changes you're suggesting there, and make a pull request back to the `exponential-backoff` branch of my fork. That way, I can see exactly what you're suggesting, without you modifying my work unilaterally.